src/storybook/story-test-runner/index.ts

```diff
@@ -21,18 +23,17 @@ function createTestplaneTests(
     { autoScreenshots, autoscreenshotSelector, autoScreenshotStorybookGlobals }: TestplaneOpts,
 ): void {
     nestedDescribe(story, () => {
-        const rawAutoScreenshotGlobalSets = {
-            ...autoScreenshotStorybookGlobals,
-            ...story.autoScreenshotStorybookGlobals,
-        };
-
+        const rawAutoScreenshotGlobalSets = extractInheritedValue(
+            story.autoScreenshotStorybookGlobals,
```

Comment:
Here `story.autoScreenshotStorybookGlobals` is `Inheritable<AutoScreenshotStorybookGlobals>`

More about `Inheritable` at `inheritable-values.ts`